src/incubator/TextField/Input.tsx

```diff
@@ -45,7 +44,7 @@ const Input = ({
 
 const styles = StyleSheet.create({
   input: {
-    flexGrow: 1,
```

Comment:
I checked why we added this flexGrow. 
We did it in this PR
https://github.com/wix/react-native-ui-lib/pull/1734

But unfortunately, I can't reproduce the bug it was meant to fix. 
We can merge the PR and worse case we can revert if the bug appear again. 

Another option is to pass a style that overrides the `flexGrow` from the ChipsInput code.

What do you prefer to do?